pages/parent-enrollment.tsx

```diff
@@ -1,82 +1,74 @@
-                    Who would you like to register for{" "}
-                    <b>
-                        Building Bridges with Music - <br />
-                        Singing Monkeys (Ages 9 and under)?{" "}
-                    </b>
-                    (select one)
-                </Text>
-            </Center>
+/**
+ * This is the page that a direct a user to register a student for a class
+ */
 
-            <Center>
-                <VStack spacing={5}>
-                    {children.map((childName) => (
-                        <Button
-                            _focus={{ boxShadow: null }}
-                            _hover={{ border: "2px solid #0C53A0" }}
-                            backgroundColor={
-                                selectedChild === childName
-                                    ? colourTheme.colors.LightGrayBlue
-                                    : "white"
-                            }
-                            lineHeight="24px"
-                            fontSize="16px"
-                            fontWeight="normal"
-                            textColor="#0C53A0"
-                            borderRadius="6px"
-                            height="50px"
-                            width="340px"
-                            key={childName}
-                            onClick={() => setSelectedChild(childName)}
-                            border={
-                                selectedChild === childName
-                                    ? null
-                                    : "2px solid #E1E1E1"
-                            }
-                        >
-                            {childName}
-                        </Button>
-                    ))}
-                </VStack>
-            </Center>
-            <Center mt="45px" mb="200px">
-                <Button
-                    height="50px"
-                    width="340px"
-                    borderRadius="6px"
-                    background={selectedChild === "" ? "#737373" : "#0C53A0"}
-                    fontWeight="normal"
-                    textColor="#FFFFFF"
-                    fontSize="16px"
-                    isDisabled={selectedChild === ""}
-                >
-                    Next
-                </Button>
-            </Center>
-        </Wrapper>
-    );
+export default function ParentEnrollClass(
+    props: ParentEnrollClassProps,
+): JSX.Element {
+    const [pageNum, setPagNum] = useState<number>(0);
+    const [selectedChild, setSelectedChild] = useState<number>(0);
+    const { user, isLoading, error } = useUser(props.session.id as string);
+
+    if (error) {
+        return <Box>{"An error has occurred: " + error.toString()}</Box>;
+    }
+    if (isLoading) {
+        return <Loading></Loading>;
+    }
+    if (user) {
+        const parentData = {
+            name: user.firstName + " " + user.lastName,
+            phone: user.parent.phoneNumber,
+        };
```

Comment:
I think you don't need this. Just pass in user as Type User. And make the according changes in the component that uses this data.
```suggestion
```

Reply:
I would do that, but the type User doesn't have the nested student and parent objects so the object user isn't of type User according to TS